lib/rack/cache/request.rb

```diff
@@ -22,7 +23,10 @@ def request_method
     def cache_control
       @cache_control ||= CacheControl.new(env['HTTP_CACHE_CONTROL'])
     end
-
+    
+    def report?
+      @env['HTTP_X_HTTP_METHOD_OVERRIDE'] == REPORT
+    end
     # True when the Cache-Control/no-cache directive is present or the
```

Comment:
Nitpick: I would like a blank line here also after the method to improve readability